applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_solver.py

```diff
@@ -283,7 +283,8 @@ def FinalizeSolutionStep(self):
             self._GetSolutionStrategy().FinalizeSolutionStep()
 
             # Limit the obtained acceleration for the next step
-            self._GetAccelerationLimitationUtility().Execute()
+            if self.main_model_part.ProcessInfo[KratosMultiphysics.STEP] > 3:
```

Comment:
Didn't we agree to deactivate this by default?

Reply:
After some test, I think it is generally necessary. Actually, if you agree, we can move it to the InitializeSolutionStep (and omit the condition on STEP).

Reply:
Then why is this change necessary? Shouldn't the check above be sufficient to avoid the buffer issues?

Reply:
"Then why is this change necessary?"
Moving this to the InitializeSolutionStep let's user observe the last (current) velocity field that has just calculated. However, from the computationa point of view it would be the same as it is now.

"Shouldn't the check above be sufficient to avoid the buffer issues?"
I am afraid there might be some cases that large (spurious) velocity variations occur at the very first steps when this is inactive. 

So, it might be safer to keep the code as it is and only move the utility to the InitializeSolutionStep.

Reply:
> "Then why is this change necessary?"
> Moving this to the InitializeSolutionStep let's user observe the last (current) velocity field that has just calculated. However, from the computationa point of view it would be the same as it is now.
> 

I'm not 100% sure since doing it in the `InitializeSolutionStep` means doing it after the clone solution step, meaning that the first and second position of the buffer have the same velocity value. Depending on how the acceleration is computed within the utility this would lead to different results. Most probably this is the reason why it was placed in the `FinalizeSolutionStep`. 

> "Shouldn't the check above be sufficient to avoid the buffer issues?"
> I am afraid there might be some cases that large (spurious) velocity variations occur at the very first steps when this is inactive.

the `if self._TimeBufferIsInitialized():` in above basically means `if self.main_model_part.ProcessInfo[KratosMultiphysics.STEP] >= 2` that is to say that the acceleration limitation utility is triggered in the first steps.

> 
> So, it might be safer to keep the code as it is and only move the utility to the InitializeSolutionStep.

This needs to be checked (see my previous comment).

Reply:
I think that even if this option is activated by default, there should be an option to switch it off. I think its not difficult to imagine situations where fluid suddenly accelerates in some part of the domain.

Reply:
Makes sense for me as well. Should we do it in this PR?

Reply:
i'd do that but I live life to the fullest

Reply:
@rubenzorrilla if you agree, we can add a line to the settings (being true by default).
"acceleration_limiter"

Reply:
Makes sense for me. Maybe we better name it `limit_acceleration` or `acceleration_cutoff` to avoid misunderstandings with the new convection limiters.